### PR TITLE
Update chapter-22.xhtml

### PR DESCRIPTION
**Pull request by legrk**
ovre -> over
remainedl -> remained

### CODE REVIEW

File: src/epub/text/chapter-22.xhtml [hunk 1]
```diff
@@ -28,7 +28,7 @@
...
+			<p>“What am I to do? To take Shamil at his word and return to him?” he thought. “He is a fox and will deceive me. Even if he did not deceive me, it would still be impossible to submit to that red liar. It is impossible⁠ ⁠… because now that I have been with the Russians he will not trust me,” thought Hadji Murád; and he remembered a Tavlinian fable about a falcon who had been caught and lived among men and afterwards returned to his own kind in the hills. He returned, wearing jesses with bells; and the other falcons would not receive him. “Fly back to where they hung those silver bells on thee!” said they. “We have no bells and no jesses.” The falcon did not want to leave his home, and remained but the other falcons did not wish to let him stay there, and pecked him to death.</p>
```
Comment (acabal) [inline src/epub/text/chapter-22.xhtml:31]: This should actually be a semicolon, not an `l`

Comment (legrk) [reply src/epub/text/chapter-22.xhtml:31]: can confirm l should be a semicolon.
Sorry, for not properly checking in scans.

Comment (acabal) [reply src/epub/text/chapter-22.xhtml:31]: Can you correct the pull request?

**Comment by acabal** [comment]
Thanks!

**Pull request merged**